geode-core/src/integrationTest/java/org/apache/geode/management/internal/cli/commands/DescribeDiskStoreCommandDUnitTest.java

```diff
@@ -30,7 +30,7 @@
 import org.apache.geode.test.junit.rules.ServerStarterRule;
 
 @Category({PersistenceTest.class})
-public class DescribeDiskStoreCommandIntegrationTest {
+public class DescribeDiskStoreCommandDUnitTest {
```

Comment:
The test is really an integration test. Tests that use `ServerStarterRule` (or `LocatorStarterRule`) don't fork separate JVMs but start a cache within the test JVM itself - this is essentially the definition of an integration test as it applies to the codebase. `ClusterStarterRule`, on the other hand, does fork separate JVMs and thus all tests that use that rule should be classified as DUnit tests.